postgresql-11.5/src/fe_utils/printbteq.c

```diff
@@ -1014,8 +1084,12 @@ print_aligned_text_bteq(const printTableContentBteq *cont, FILE *fout, bool is_p
                         fputs(format->wrap_left, fout);
                     else if (wrap[j] == PRINT_LINE_WRAP_NEWLINE_BTEQ)
                         fputs(format->nl_left, fout);
-                    else
-                        fputc(' ', fout);
+                    else {
+                        if (j + 1 == col_count_copy && table_width2 == 0) {}
```

Comment:
reverse the conditional statement to not have an empty block.

Reply:
done as specified